lib/modules/platform/github/index.spec.ts

```diff
@@ -210,6 +210,34 @@ describe('modules/platform/github/index', () => {
       const repos = await github.getRepos();
       expect(repos).toStrictEqual(['a/b', 'c/d']);
     });
+
+    it('should return an array of repos when using GitHub App Installation Token', async () => {
```

Comment:
Is this existing code which was previously untested?

Reply:
I initially added this test in the first commit to get the coverage back to 100%. I needed a case where the token starts with `ghs_`.
This test doesn't make any sense right now. Instead, I need a test for `lib/util/http/auth.ts` since 4f203bec53ddeee0eea6ae507980c0c5d2bb19e1 to get back to 100%.